lib/vanagon/platform.rb

```diff
@@ -226,7 +230,7 @@ def is_unix?
     #
     # @return [true, false] true if it is a windows variety, false otherwise
     def is_windows?
-      return !!@name.match(/^win-.*$/)
+      return !!@name.match(/^windows-.*$/)
```

Comment:
I'm not changing this here. That should be something we change in the main branch and not a feature branch.